pkg/common/cns-lib/volume/manager.go

```diff
@@ -332,15 +329,20 @@ func (m *defaultManager) ResetManager(ctx context.Context, vcenter *cnsvsphere.V
 	managerInstanceLock.Lock()
 	defer managerInstanceLock.Unlock()
 	log.Infof("Re-initializing defaultManager.virtualCenter")
+	existingVCInstance := managerInstance.virtualCenter
 	managerInstance.virtualCenter = vcenter
 	if m.tasksListViewEnabled {
 		m.listViewIf.SetVirtualCenter(ctx, managerInstance.virtualCenter)
 	}
 	if m.virtualCenter.Client != nil {
-		m.virtualCenter.Client.Timeout = time.Duration(vcenter.Config.VCClientTimeout) * time.Minute
+		m.virtualCenter.Client.Timeout = noTimeout
 		log.Infof("VC client timeout is set to %v", m.virtualCenter.Client.Timeout)
 	}
 	log.Infof("Done resetting volume.defaultManager")
+	err := existingVCInstance.Disconnect(ctx)
+	if err != nil {
+		log.Errorf("Error disconnecting older vc session. err: %v", err)
+	}
```

Comment:
@divyenpatel, can you please also review this section. Here I am disconnecting the older session after a reload config event.